src/main/java/org/jahia/modules/sitemap/filter/SitemapCacheFilter.java

```diff
@@ -60,9 +58,8 @@
 public class SitemapCacheFilter extends AbstractFilter {
 
     private static final Logger logger = LoggerFactory.getLogger(SitemapCacheFilter.class);
-
-    private static final long EXPIRATION = ConfigServiceUtils.getCacheDuration();
-
+    private static final String SITEMAP = "sitemap";
```

Comment:
I might've missed but I don't see this constant used anywhere? If it isn't then we should remove.

Reply:
Thanks for the catch removed